Apps/frontend/src/views/AmplitudeSlider.svelte

```diff
@@ -0,0 +1,27 @@
+<script>
+  import { amplitudeAdjustment } from "../stores.js";
+  import { MIN_AMPLITUDE, MAX_AMPLITUDE } from "../const.js";
+  export let onInput = (amplitude) => {
+    return amplitude;
+  };
+
+  export let channel;
+  // default amplification
+  let amplitude = 1.0;
+</script>
+
+<div>
+  <input
+    type="range"
+    bind:value={amplitude}
+    on:input={() => {
+      $amplitudeAdjustment[channel] = amplitude;
+      onInput(amplitude);
+    }}
+    min={MIN_AMPLITUDE}
+    max={MAX_AMPLITUDE}
+    step="0.01"
+    class="slider"
```

Comment:
Why this `class` attribute?

Reply:
others sliders have the same attribute. idea was, that if we query or style sliders we have a general class attribute. should i remove it?

Reply:
Yeah, I see that the others have it as well. I think keep it as it is for now, it doesn't really hurt.